vendor/assets/javascripts/changed_form_modal.js

```diff
@@ -1,18 +1,31 @@
 var NR = Cmsify.NestedResource.prototype;
+var filterAuthenticityToken = Cmsify.util.filterAuthenticityToken;
 var CFM = Cmsify.ChangedFormModal = function(el) {
-  this.$el = $(el);
-  this.form = this.$el.find('form');
-  this.modal = UIkit.modal(this.$el.find('.js-unsaved-modal'));
-  this.$el.find('.js-check-changed').each(function(index, ele) {
-    $(ele).on('click', function(e) {
-      if (this.setIsModified()) {
-        e.preventDefault();
-        this.modal.show();
-      }
-    }.bind(this));
+  this.$el = $('form');
+  if(!this.$el.length);
+  this.lastData = this.serializeFields();  
+  $('a').on('click', function(event) {
+    if ((typeof(event.target.dataset['ukModal']) !== 'undefined') || $(event.target).hasClass('uk-modal-close')) return;
+    if (this.setIsModified()) {
+      this.modalOpened = true;
```

Comment:
Can you change the boolean to `this.isWarningModelOpened`. I wasn't immediately sure which `model` this was referring to. Also move everything inside the `if (this.setIsModified())` into it's own method called like `this.openWarningModel()`. Just so it's a bit easier to read at a glance.